pages/testlibrary.tsx

```diff
@@ -20,24 +20,21 @@ const TestLibrary: NextPage<TestlibraryProps> = ({candidateTestStatus}) => {
             <Head>
                 <title>TestSwitch Test Library</title>
             </Head>
-            <TestLibraryStepper key={key} candidateTestStatus={candidateTestStatus}/>
+            <TestLibraryStepper key={key} candidateTestStatuses={sessionCandidate.testStatuses}/>
         </Layout>
     );
 };
 
-//TODO placeholder id for candidateID, need to know how candidate id is paired with session
-const candidateId = () => {
-    return 1;
-};
+//TODO placeholder id for candidateID,until url has token as query
+const candidateToken = `815b47a8-0b7a-4d6a-99e9-2fe130c5b774`;
 
 export const getServerSideProps: GetServerSideProps = async context => {
-    const tests = getCandidateTests();
-    const results = getCandidateTestResults();
+    const token = context.query.token as string;
+    const sessionData = getSessionCandidate(candidateToken);
```

Comment:
could we use `token` here rather than `candidateToken`?

Reply:
yep. just for testing. Now cleared out. ✔️